frontend/src/scenes/events/EventsTable.js

```diff
@@ -190,10 +160,110 @@ function _EventsTable({ fixedFilters, filtersEnabled = true, pageKey }) {
             },
         },
     ]
+    const otherEventProperties = eventProperties.filter(
+        (e) => defaultColumns.find((d) => d.eventProperties && d.eventProperties.includes(e.value)) === undefined
+    )
+    const availableConfigsOptions = [
+        ...defaultColumns.map(tableColumnToCheckboxOption),
+        ...otherEventProperties.map(eventPropertyToCheckboxOption),
+    ]
+    let selectedConfigOptions = columnConfig === 'DEFAULT' ? defaultColumns.map((e) => e.key) : columnConfig
+
+    const columnsToRenderFromConfig =
+        columnConfig === 'DEFAULT'
+            ? defaultColumns
+            : columnConfig.map(
+                  (e) =>
+                      defaultColumns.find((d) => d.key === e) || {
+                          title: keyMapping['event'][e] ? keyMapping['event'][e].label : e,
+                          key: e,
+                          render: function renderURL({ event }) {
+                              if (!event) {
+                                  return { props: { colSpan: 0 } }
+                              }
+                              if (filtersEnabled) {
+                                  return (
+                                      <FilterPropertyLink
+                                          className={'ph-no-capture ' + rrwebBlockClass}
+                                          property={e}
+                                          value={event.properties[e]}
+                                          filters={{ properties }}
+                                      />
+                                  )
+                              }
+                              return <Property value={event.properties[e]} />
+                          },
+                          ellipsis: true,
+                      }
+              )
+    let columns = [
+        {
+            title: `Event${eventFilter ? ` (${eventFilter})` : ''}`,
+            key: 'event',
+            fixed: 'left',
```

Comment:
I think the scrolling looks better if this is removed.

Reply:
Removed. Amending the commit with other fixes. Though I am of opinion event name should always be visible so that when a person scroll to right most column for a very wide table they don't confuse for which event name do the value belong to.

Reply:
fixed in aca8c1e5d652499df7bc0c8c5f9528d5ecfc22c3